### ISSUE TITLE
Can't build dino beacuse ./configure error

### ISSUE THREAD

**Issue by DarkoDarkShadow** [dino/dino#1261]
Hello, I'm trying to build dino for Debian 11 (Bullseye) but I get the following error after I run "./configure":

```
-- Configuring Dino  with plugins: http-files, ice, omemo, openpgp, rtp
-- ALPN support enabled
CMake Error at /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find GTK4 (missing: GTK4_LIBRARY)
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:458 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindGTK4.cmake:28 (find_package_handle_standard_args)
  main/CMakeLists.txt:6 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/darko/Downloads/dino-master/build/CMakeFiles/CMakeOutput.log".

```
I can't find any GTK4 package for my current Debian verison in order to solve the missing library. Of course I installed all dependencies that appears here: [](https://github.com/dino/dino/wiki/Build#dependencies)

Here's my "CMakeOutput.log" file:
[CMakeOutput.log](https://github.com/dino/dino/files/9388197/CMakeOutput.log)

**Comment by licaon-kter** [comment]
You need packages from Testing: https://tracker.debian.org/pkg/gtk4

**Comment by Ke1i** [comment]
Had something similar happening to me on LMDE 5.
Try running `git checkout v0.3.1` after cloning, then compile. This is what worked for me.

**Comment by fiaxh** [comment]
The current Dino master depends on GTK4, which is not part of the Debian Stable repos.

There is a repo for Debian 11 by @mar-v-in that contains Dino and all dependencies that are not part of Debian 11 (GTK4, pango, libadwaita, wayland-protocols). If you install Dino from that repo, it will also install all dependencies. Or you can only install the dependencies from that repo and continue to compile Dino from source if you want.

The repo is here: https://software.opensuse.org//download.html?project=home%3Alarma&package=dino
And here is a more user-friendly entry page for the Dino package: https://software.opensuse.org//download.html?project=home%3Alarma&package=dino
